packages/core-sdk/src/types/resources/dispute.ts

```diff
@@ -114,3 +114,30 @@ export type DisputeAssertionRequest = {
    */
   wipOptions?: Omit<WipOptions, "useMulticallWhenPossible">;
 } & WithTxOptions;
+
+/**
+ * Tags refer to the “labels” that can be applied to IP Assets in the protocol when raising a dispute.
+ * Tags must be whitelisted by protocol governance to be used in a dispute.
+ * @see {@link https://docs.story.foundation/docs/dispute-module#dispute-tags | Dispute Tags}
+ */
+export enum DisputeTargetTag {
+  /** Refers to registration of IP that already exists. */
+  IMPROPER_REGISTRATION = "IMPROPER_REGISTRATION",
+  /**
+   * Refers to improper use of an IP Asset across multiple items.
+   * For more details, see the {@link https://docs.story.foundation/concepts/programmable-ip-license/overview | Programmable IP License (PIL)} documentation.
+   */
+  IMPROPER_USAGE = "IMPROPER_USAGE",
+  /** Refers to missing payments associated with an IP. */
+  IMPROPER_PAYMENT = "IMPROPER_PAYMENT",
+  /**
+   * Refers to “No-Hate”, “Suitable-for-All-Ages”, “No-Drugs-or-Weapons” and “No-Pornography”.
+   * These items can be found in more detail in the {@link https://docs.story.foundation/concepts/programmable-ip-license/overview  | 💊 Programmable IP License (PIL) } legal document.
```

Comment:
There is some lack of consistency in how links in this documentation are written.  Sometimes it says `@see { @link`, other `see the {@link` and other just `{@link`